src/index.js

```diff
@@ -33,3 +33,4 @@ const extensions = {
     },
 };
 export const LT = { ...app, ...items, ...activity, ...player, ...questions, ...sections, ...diagnostics, ...logger, ...extensions };
+window.LT = LT;
```

Comment:
We don't add to the global scope here. Instead, we give the calling page the option to store it wherever they want. This is a better approach for securing access to LT.